azurelinuxagent/ga/exthandlers.py

```diff
@@ -182,21 +191,23 @@ def get_exthandlers_handler():
 class ExtHandlersHandler(object):
     def __init__(self):
         self.protocol_util = get_protocol_util()
-        self.protocol = None
+        self.protocol = self.protocol_util.get_protocol()
         self.ext_handlers = None
+        # self.remote_access = None
         self.last_etag = None
         self.last_upgrade_guids = {}
         self.log_report = False
         self.log_etag = True
         self.log_process = False
-
+        self.osUtil = get_osutil()
```

Comment:
There should be no distro-specific logic in here. The extension handler mechanism is agnostic.